src/app/services/Scheduler.ts

```diff
@@ -0,0 +1,12 @@
+export const Scheduler = (interval = 300000) => ({
```

Comment:
Functions should always be `camelCase` and class `PascalClass`.

Reply:
Do you think that we should create a class instead of just a function in this case to explicit that the Scheduler should be `service/module` ? 🤔